src/FilterBundle/Helper/EnumAttributeHelper.php

```diff
@@ -47,41 +70,51 @@ public function __construct(Client $client)
      */
     public function getLabels(string $attributeName): array
     {
-        $productTypeRequest = ProductTypeQueryRequest::of();
+        $cacheItem = $this->cacheItemPool->getItem(self::CACHE_KEY . '-' . md5($attributeName));
 
-        $response = $this->client->execute($productTypeRequest);
+        if ($cacheItem->isHit()) {
+            $labels = $cacheItem->get();
+        } else {
+            $productTypeRequest = ProductTypeQueryRequest::of();
```

Comment:
I think you're right. But this is not a "problem" of this pr. I'll fix it in another ticket.